package/CHANGELOG

```diff
@@ -84,6 +84,8 @@ Fixes
   * Fix syntax warning over comparison of literals using is (Issue #3066)
 
 Enhancements
+  * Introduce possibility for setting box dimensions for all timesteps in 
```

Comment:
Could you please be clearer in this update? Specifically mention transformations.